handbook/direction/maturity/index.md

```diff
@@ -0,0 +1,63 @@
+# Feature Maturity Tracker
+
+Sourcegraph keeps track of how mature each of our feature areas are, to help you make the best decisions about how to use our product. Clicking on the feature name will take you to its documentation.
+
+If you do not see content below, check you have JavaScript enabled then contact Product Management.
+
+## Maturity levels
+
+- ○: not yet available, but on our roadmap
+- ◔: available in the product, but may not be ready for production use yet
+- ◑: usable for primary jobs to be done, but may lack depth
+- ●: a complete feature used extensively by our customers
+- ♥: a lovable, complete, and market leading experience
+
+<script>
```

Comment:
I wonder if this generation code is worth it compared to just maintaining the "result" in Markdown? It seems more approachable to edit the markdown directly or look at the version history (don't need to find the JSON file to look at).
A JSON file with generator code can enforce consistency, but if there are some examples already present folks would probably intuitively follow the same format so I wouldn't be too worried about that.

Reply:
Yeah, that's a good question mentioned in the description. The benefit of having content driven by data files is that it can be used in multiple places without having to update multiple places. This would be the first place this was used, but you could imagine info on maturity showing up in other places too.